docs/source/troubleshooting/logs.rst

```diff
@@ -95,19 +95,15 @@ The following list outlines some of the common logs and what methods are recomme
 Note that when writing to these logs that it’s important to be aware of how they behave.
 For example, by default the ``errlog.txt`` file will persist across reboots but be rotated by ``logrotate`` very quickly due to file size while the ``messages`` log will not persist across reboots but will be rotated less often.
 
-Enabling Persistent Logs
-========================
-By default, logs are stored in ``/var/log`` which is a symlink to ``/var/volatile/log``.
-The ``/var/volatile`` location is a RAM-based tmpfs filesystem whose contents are lost on each reboot.
-This section explains how to make those logs persistent.
```

Comment:
Instead of removing this correct context about how logging worked on older versions, lets move it to live beneath a section talking about logging prior to version 11.2.

Reply:
The info is still there, just move a few lines below - GitHub UI doesn't make it very clear.